expyfun/_utils.py

```diff
@@ -592,13 +595,16 @@ def set_config(key, value):
 
     Parameters
     ----------
-    key : str
-        The preference key to set.
+    key : str | None
+        The preference key to set. If None, a tuple of the valid
+        keys is returned, and ``value`` is ignored.
     value : str |  None
         The value to assign to the preference key. If None, the key is
         deleted.
     """
 
+    if key is None:
+        return known_config_types
```

Comment:
now that I think about it we might as well return `sorted(known_config_types)` to be a little bit nicer to users